slick/src/main/scala/slick/dbio/DBIOAction.scala

```diff
@@ -176,19 +177,19 @@ object DBIOAction {
   }
 
   /** Transform a `TraversableOnce[ DBIO[R] ]` into a `DBIO[ TraversableOnce[R] ]`. */
-  def sequence[R, M[+_] <: TraversableOnce[_], E <: Effect](in: M[DBIOAction[R, NoStream, E]])(implicit cbf: CanBuildFrom[M[DBIOAction[R, NoStream, E]], R, M[R]]): DBIOAction[M[R], NoStream, E] = {
+  def sequence[R, M[+_] <: IterableOnce[_], E <: Effect](in: M[DBIOAction[R, NoStream, E]])(implicit cbf: BuildFrom[M[DBIOAction[R, NoStream, E]], R, M[R]]): DBIOAction[M[R], NoStream, E] = {
```

Comment:
To keep this binary compatible the implicit db must be of type `Factory[R, M[R]]` (which expands to `CanBuildFrom[Nothing, R, M[R]]` on scala. 2.12

The consequence of this is that on the lines below, you must call  `cbf.newBuilder` instead of `cbf.newBuilder(in)`
I am not completely sure what the consequences are...

Reply:
I copied over the DBIOAction that you had in your PR.